nodejs/listeners/actions/block_open_task_list_home.js

```diff
@@ -0,0 +1,15 @@
+const { Task } = require('../../models');
+
+module.exports = (app) => {
+  app.action({ action_id: 'openTaskListHome', type: 'block_actions' }, async ({ ack, action }) => {
+    await ack();
+    const tasksToUpdate = [];
+    if (action.selected_options.length > 0) {
+      action.selected_options.forEach(async (option) => {
+        const taskID = option.value.split('-')[2];
+        tasksToUpdate.push(taskID);
+      });
+      Task.update({ status: 'CLOSED' }, { where: { id: tasksToUpdate } });
+    }
+  });
```

Comment:
@kian2attari - You first implemented this pattern - Any suggestions on how to run it through `jest` to get some test coverage?

Reply:
Hi @colmdoyle , I'm a fan of the convention used [here](https://github.com/AmericanAirlines/AskingForAFriend/tree/main/src/tests/slack/shortcuts) for testing the shortcut listener, I think it'd work well

Reply:
Ha, great minds think alike, I spun up a branch this morning to implement exactly that pattern!